plugins/source/k8s/resources/services/apps/replica_sets_fetch.go

```diff
@@ -0,0 +1,43 @@
+package apps
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/k8s/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ====================================================================================================================
```

Comment:
can we remove all those comments from all files? (search and replace I guess will work)